bin/update/deploy.py

```diff
@@ -36,8 +36,6 @@ def update_info(ctx):
         ctx.local('git status')
         ctx.local('git submodule status')
 
-        ctx.local('git rev-parse HEAD > static/revision')
```

Comment:
With this removal you might want to update the docstring to say that it's just outputting info instead of writing to a file.